backend/src/main/java/com/benine/backend/http/CameraInfoHandler.java

```diff
@@ -28,11 +27,6 @@ public CameraInfoHandler(Logger logger) {
   public void handle(HttpExchange exchange) throws IOException {
     getLogger().log("Got an http request with uri: "
             + exchange.getRequestURI(), LogEvent.Type.INFO);
-    try {
-      respond(exchange, getCameraController().getCamerasJSON());
-    } catch (CameraConnectionException e) {
-      getLogger().log("Cannot connect to camera with id: " + e.getCamId(), LogEvent.Type.CRITICAL);
-    }
-    respond(exchange, "{\"succes\":false}");
+    respond(exchange, getCameraController().getCamerasJSON());    
```

Comment:
That is true but when there is only one camera unreachable, I think the others should still be usable. So previously when the getCamerasJSON fails it means at least one camera fails, but the information of the other cameras should still be sent to the client.